### DISCUSSION TITLE
Installation Problem, httpd fatal error and cant find perl file

### DISCUSSION THREAD

**Discussion by KitaBiserov** [juanluisbaptiste/docker-otrs#124] [Q&A]
![image](https://user-images.githubusercontent.com/72317821/186373036-2c23146e-e631-4077-b04e-3ad857f5d71c.png)

**I followed the github installation instructions and faced with this problems. 
I have same questions about it:

1.  What is SERVER NAME exactly means ? Is it SMTP server ?
2. How can I add the /opt/otrs/scripts/apache2-perl-startup.pl  file?
3. How can I resolve next problem : ** 

> [perl:error] [pid 39] Can't locate Apache/DBI.pm in @INC (@INC contains: /opt/otrs/Custom /opt/otrs/Kernel/cpan-lib /opt/otrs/ /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 . /etc/httpd) at /opt/otrs/scripts/apache2-perl-startup.pl line 52.\nBEGIN failed--compilation aborted at /opt/otrs/scripts/apache2-perl-startup.pl line 52.\nCompilation failed in require at (eval 2) line 1.\n

Thank you!

**Comment by abscher** [comment]
exactly the same problem

**Reply by alvherrey** [reply]
the same,
the problem was with de /opt/otrs/Kernel volume mount, it was empty dir, i umounted and is working.
